out/extension.js

```diff
@@ -0,0 +1,60 @@
+"use strict"
```

Comment:
Files in `out` should not be tracked by git.
Let's make sure to put `out` in `.gitignore`.